code/__HELPERS/bluespace_entropy.dm

```diff
@@ -0,0 +1,107 @@
+GLOBAL_VAR_INIT(bluespace_entropy, 0)
+
+/proc/go_to_bluespace(turf/T, entropy=1, minor_dristortion=FALSE, ateleatom, adestination, aprecision=0, afteleport=1, aeffectin=null, aeffectout=null, asoundin=null, asoundout=null)
+	bluespace_entropy(entropy, T, minor_dristortion)
+	do_teleport(ateleatom, adestination, aprecision, afteleport, aeffectin, aeffectout, asoundin, asoundout)
+
+/proc/bluespace_entropy(max_value=1, turf/T, minor_dristortion=FALSE)
+	if(minor_dristortion && prob(1))
+		GLOB.bluespace_entropy -= rand(25, 100)
+		bluespace_distorsion(T, minor_dristortion)
+	else
+		GLOB.bluespace_entropy += rand(0, max_value)
+		var/entropy_cap = rand(100, 300)
+		if(GLOB.bluespace_entropy >= entropy_cap)
+			bluespace_distorsion(T, minor_dristortion)
+			GLOB.bluespace_entropy -= entropy_cap
+
+/proc/bluespace_distorsion(turf/T, minor_dristortion=FALSE)
+	var/distortion_value = rand(1, 100)
+	switch(distortion_value)
```

Comment:
Bluespace crystals were added recently, so you can add it here.

Reply:
These already contribute to increasing entropy when used to teleport something.

Reply:
I mean, they are blue space related event.